src/src/com/microsoft/aad/adal/StorageHelper.java

```diff
@@ -127,7 +127,7 @@
 
     private static String sBlobVersion;
```

Comment:
I'm not worrying about this the version too much at this moment. The problem with the current static key and mackey is we have a early return when starting doing the encryption, and it's causing failure for broker apps. i don't get your concern for failing to get key for one instance but not the other instance.